### ISSUE TITLE
Training encountered a problem

### ISSUE THREAD

**Issue by ddddkk** [UX-Decoder/Segment-Everything-Everywhere-All-At-Once#79]
Thank you very much for making the training code for SEEM publicly available. However, I encountered several issues during my training process. Could you please provide more details on dataset preparation? Is it sufficient to place the dataset in COCO format into the .xdecoder_data folder? Or are there other specific steps required? I noticed that the instructions mention the 'Prepare panoptic_train2017, panoptic_semseg_train2017' folders, but I didn't find these folders in the COCO dataset I downloaded. Did I miss a crucial step? I look forward to your response and am very grateful!

非常感谢您公开了SEEM的训练代码，但是在我训练过程中我遇到了诸多问题。首先是数据集的准备可以再说的详细一点？只要把COCO格式的数据集放到.xdecoder_data文件中去就可以了吗？还是需要其他具体的操作？我看到说明中有：Prepare panoptic_train2017, panoptic_semseg_train2017这两个文件夹，但是我下载到的COCO数据集都没有这两个文件夹。是我漏掉了某个关键步骤吗？期待您的解答，不胜感激！

Running Error：
INFO:trainer.distributed_trainer:Using CUDA
WARNING:trainer.utils.mpi_adapter:----------------
WARNING:trainer.utils.mpi_adapter:MPI Adapter data
WARNING:trainer.utils.mpi_adapter:----------------
WARNING:trainer.utils.mpi_adapter:environment info: no MPI
WARNING:trainer.utils.mpi_adapter:init method url: tcp://127.0.0.1:36873
WARNING:trainer.utils.mpi_adapter:world size: 1
WARNING:trainer.utils.mpi_adapter:local size: 1
WARNING:trainer.utils.mpi_adapter:rank: 0
WARNING:trainer.utils.mpi_adapter:local rank: 0
WARNING:trainer.utils.mpi_adapter:master address: 127.0.0.1
WARNING:trainer.utils.mpi_adapter:master port: 36873
WARNING:trainer.utils.mpi_adapter:----------------
INFO:trainer.distributed_trainer:Save config file to ../../data/output/test/conf_copy.yaml
INFO:trainer.distributed_trainer:Base learning rate: 0.0001
INFO:trainer.distributed_trainer:Number of GPUs: 1
INFO:trainer.distributed_trainer:Gradient accumulation steps: 1
INFO:trainer.default_trainer:Imported base_dir at base_path ./
Deformable Transformer Encoder is not available.
WARNING:datasets.registration.register_vlp_datasets:WARNING: Cannot find VLPreDataset. Make sure datasets are accessible if you want to use them for training or evaluation.
WARNING:datasets.registration.register_vlp_datasets:WARNING: Cannot find VLPreDataset. Make sure datasets are accessible if you want to use them for training or evaluation.
WARNING:datasets.registration.register_vlp_datasets:WARNING: Cannot find VLPreDataset. Make sure datasets are accessible if you want to use them for training or evaluation.
WARNING:datasets.registration.register_vlp_datasets:WARNING: Cannot find VLPreDataset. Make sure datasets are accessible if you want to use them for training or evaluation.
WARNING:datasets.registration.register_vlp_datasets:WARNING: Cannot find VLPreDataset. Make sure datasets are accessible if you want to use them for training or evaluation.

**Comment by ddddkk** [comment]
I just came into contact with AI. If I have any silly questions, please help me point them out，thanks ：）

**Comment by MaureenZOU** [comment]
1. 你没有compile deformable transformer但是这个跟dataset没有关系。
2. COCO dataset的preparation要refer to mask2former，可以详细参考https://github.com/UX-Decoder/Segment-Everything-Everywhere-All-At-Once/blob/v1.0/assets/readmes/DATASET.md#coco-seem--x-decoder 看第一行注释

**Comment by Note-Liu** [comment]
> 1. compile deformable transformer

How to compile deformable transformer?
